### ISSUE TITLE
Keep folder structure in Postman?

### ISSUE THREAD

**Issue by ghost** [fbenz/restdocs-to-postman#5]
Thank you very much for this project! :)

I'm wondering if it would be possible to keep the folder structure that I have for my generated snippets in Postman?

For example, I have these generated snippets:
![screen shot 2018-01-11 at 14 48 52](https://user-images.githubusercontent.com/32162187/34828662-907069e0-f6de-11e7-8a90-41547ee546db.png)

But after using restdocs-to-postman and opening the collection in Postman I see this:
![screen shot 2018-01-11 at 14 52 42](https://user-images.githubusercontent.com/32162187/34828843-230010bc-f6df-11e7-84bc-be3a1f10235a.png)

i.e. all the requests are in the same folder.

**Comment by fbenz** [comment]
Good idea. Grouping into folders could be added as an option. Insomnia also supports folders. This helps because the flow is cURL -> Insomnia -> Postman. But still this flow has to be touched as all cURL commands are concatenated and delivered as one input to the Insomnia converter. Thus, this part has to be changed quite a bit. For example, by converting each cURL command to the Insomnia format and then combining all of them together with the information about the folders.

The next question is about extracting the folder from the file path. `restdocs-to-postman` does not require that the path contains the folder `generated-snippets` and thus one should not rely on this. A common pattern is `.../{group}/{request}/curl-request.adoc` and thus one could extract `{group}` as a folder name if it exists. All requests that can not be put into a folder can either stay on the top level (preferred) or can be put in a special folder.

So this feature is possible but requires a bit of thought and work. Going a bit further, I was also thinking about how to update existing collections after they have been modified manually (fuzzy search for the right place to update). But this is yet another problem and definitely a complex one.

**Comment by fbenz** [comment]
Support for sorting requests into folders is included in the 2.0.0 release. Currently, there is only one option to turn the file path into a folder name: `secondLastFolder`. This one does exactly what you proposed and what should be helpful for most REST Docs projects. However, it's simple to add additional functions that turn file paths and URLs into folder names.

**Comment by ghost** [comment]
Thank you @fbenz!

I'm not sure if I'm doing it right, though. Here's what I'm doing:
```
➜  target git:(master) restdocs-to-postman --version
2.0.0
➜  target git:(master) restdocs-to-postman --input generated-snippets --export-format postman --determine-folder secondLastFolder --output postman-collection.json
➜  target git:(master) ll generated-snippets 
total 48K
drwxrwxr-x 12 stian stian 4,0K jan.  20 10:35 .
drwxrwxr-x 12 stian stian 4,0K jan.  20 10:39 ..
drwxrwxr-x  5 stian stian 4,0K jan.  20 10:34 assets
drwxrwxr-x  4 stian stian 4,0K jan.  20 10:34 events
drwxrwxr-x  3 stian stian 4,0K jan.  20 10:34 eventtransformers
drwxrwxr-x  5 stian stian 4,0K jan.  20 10:34 login
drwxrwxr-x  4 stian stian 4,0K jan.  20 10:34 projects
drwxrwxr-x  5 stian stian 4,0K jan.  20 10:34 raw
drwxrwxr-x  3 stian stian 4,0K jan.  20 10:35 semantictypes
drwxrwxr-x  3 stian stian 4,0K jan.  20 10:34 shiro
drwxrwxr-x  3 stian stian 4,0K jan.  20 10:34 storage
drwxrwxr-x  7 stian stian 4,0K jan.  20 10:35 timeseries
```
And when I open that `postman-collection.json` file it looks like this in postman:
![screenshot from 2018-01-20 10-42-02](https://user-images.githubusercontent.com/32162187/35182095-913347de-fdce-11e7-9315-b17c9c669dad.png)

I was expecting to see the folders I have in `target/generated-snippets` there. Am I doing something wrong?

**Comment by fbenz** [comment]
You are doing everything right, but I introduced a bug when I renamed the CLI option at the end. It's fixed in 2.0.1.

**Comment by ghost** [comment]
Very nice! I can confirm that 2.0.1 works well :)
